src/styles/blocks/product.scss

```diff
@@ -0,0 +1,70 @@
+.product {
+  display: flex;
+  flex-direction: column;
+  color: $main-font-color;
+
+  &__image {
+    background-image: url(/src/images/imac.jpeg);
+    height: 134px;
+    width: 160px;
+    background-size: 100% 100%;
+    margin-bottom: 40px;
+    margin-inline: auto;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    margin-block: 0 4px;
+  }
+
+  &__code {
+    color: $second-font-color;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    align-items: flex-end;
+  }
+
+  &__stars {
+    display: flex;
+  }
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+      background-image: url(/src/images/star-active.svg);
```

Comment:
The `background-image` should be consistent for all `.product__star` elements, and the active state should be controlled with a modifier class. This will make it easier to manage active vs. default stars, and it's more in line with BEM methodology.